web/src/components/Upload_Confirmation/index.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState, useEffect } from 'react'
+import {
+  Message,
+  Icon,
+  Button,
+  Container,
+  Input,
+  Dropdown
+} from 'semantic-ui-react'
+import styled from 'theme'
+
+const StyledContainer = styled(Container)`
+  margin-top: 30px;
+`
+
+const ContentContainer = styled.div`
+  margin: 0px 130px 0px 130px;
+`
+
+const SuccessBanner = styled(Message)`
+  background: #619e54 !important;
+  height: 48px;
+  width: 100%;
+  display: flex;
+  border-radius: 4px;
+  align-items: center;
+  margin-bottom: 32px;
+  * {
+    color: ${({ theme }) => theme.colors.white};
+  }
+  i.icon {
+    font-size: 20px;
+  }
+  h3 {
+    font-size: 24px;
+  }
+`
+
+const SuccessIcon = () => <Icon name='check circle outline' />
+
+const SuccessMessage = styled.span`
+  font-family: ${({ theme }) => theme.fonts.secondary};
+  font-size: 1rem;
+`
+
+const FormMessage = styled.h3`
+  font-size: 1.5rem;
+`
+
+const FormContainer = styled.div`
+  width: 500px;
+  margin: 0 auto;
+`
+
+const FormContent = styled.div`
+  padding: 1.85rem 4.5rem 5rem 2rem;
+`
+const DisabledInput = styled(Input)`
+  min-width: auto;
+  width: 160px;
+`
+
+const CalendarIcon = () => <Icon name='calendar outline' />
+
+const DropdownInput = styled(Dropdown)`
+  min-width: auto !important;
+  width: 160px;
+  //eventually need to make borders of this dropdown #232735 per wireframe
```

Comment:
It's cool. We're still iterating on getting the code to match the wireframes. We can fix it in a future push.